src/app/core/services/mocks/mock.auth.service.ts

```diff
@@ -81,6 +77,18 @@ export class MockAuthService {
     this.authStateSource.next(newAuthState);
   }
 
+  setTitleWithPhaseDetail(): void {
+    const appSetting = require('../../../../../package.json');
+    const title = `${appSetting.name} ${appSetting.version} - ${this.phaseService.getPhaseDetail()}`;
+    this.titleService.setTitle(title);
+  }
+
+  setLandingPageTitle(): void {
+    const appSetting = require('../../../../../package.json');
```

Comment:
Just a question here actually! Do we normally use relative or absolute paths for these kind of common files like `package.json` o.o

Reply:
@seanlowjk 

Hmm so far it has been relative links. But we can look into defining an absolute path for package json to make it cleaner but its only used in about 2/3 places I think. Also to define an absolute path we'd need to modify the tsconfig?

Reply:
Ahhhhh I see..... 

Then I think as of now, we can just stick to relative paths :P

Reply:
So far, I've only seen relative paths used in this context. Is there a benefit to using absolute paths?